beacon-chain/core/electra/upgrade.go

```diff
@@ -15,6 +15,129 @@ import (
 	"github.com/pkg/errors"
 )
 
+// ConvertToElectra converts a Deneb beacon state to an Electra beacon state. It does not perform any fork logic.
+func ConvertToElectra(beaconState state.BeaconState) (state.BeaconState, error) {
```

Comment:
is it not worth it to use `ConvertToElectra` in `UpgradeToElectra` like what you did for Altair?

Reply:
Fixed in [bf1ea5c](https://github.com/OffchainLabs/prysm/pull/15250/commits/bf1ea5cc40c43fa6c5ae43977ef7d8a084758374). Please notice that this is not straightforward as in Altair and it requires a copy of the validator slice making the Electra Fork slower.